src/components/TimelineList/TimelineList.js

```diff
@@ -0,0 +1,412 @@
+    onHover: (state) => handleTemporalMarkerHover('end', state)
+  })
+
+  if (!timelineWrapperRef.current) return null
+
+  const focusedRangeStyle = {}
+  const focusedRangeMaskLeftStyle = {}
+  const focusedRangeMaskRightStyle = {}
+  const temporalStartStyle = {}
+  const temporalEndStyle = {}
+  const temporalRangeStyle = {}
+  const temporalRangeMouseOverStyle = {}
+
+  // If a position is set for the temporal selection mouseover indicator, set the left property to that value
+  if (temporalRangeMouseOverPosition) {
+    temporalRangeMouseOverStyle.left = temporalRangeMouseOverPosition
+  }
+
+  const { end, start } = temporalRange
+  const { end: focusedIntervalEnd, start: focusedIntervalStart } = focusedInterval
+
+  // If a time is set for the start and end, set the left property to that temporal position
+  const timelineWrapperWidth = timelineWrapperRef.current.getBoundingClientRect().width
+
+  if (start) {
+    temporalStartStyle.left = (
+      getPositionByTimestamp({
+        timestamp: start,
+        timeIntervals,
+        zoomLevel,
+        wrapperWidth: timelineWrapperWidth
+      })
+    )
+  }
+
+  if (end) {
+    temporalEndStyle.left = (
+      getPositionByTimestamp({
+        timestamp: end,
+        timeIntervals,
+        zoomLevel,
+        wrapperWidth: timelineWrapperWidth
+      })
+    )
+  }
+
+  if (end || start) {
+    const range = [
+      start || timeIntervals[0],
+      end || generateEndTime(timeIntervals, zoomLevel)
+    ]
+
+    const [startPosition, endPosition] = range
+
+    // Style the temporal range area
+    temporalRangeStyle.left = getPositionByTimestamp({
+      timestamp: startPosition,
+      timeIntervals,
+      zoomLevel,
+      wrapperWidth: timelineWrapperWidth
+    })
+    temporalRangeStyle.width = determineScaledWidth(
+      endPosition - startPosition,
+      zoomLevel,
+      timelineWrapperWidth
+    )
+  }
+
+  if (focusedIntervalStart && focusedIntervalEnd) {
+    // Style the focused area
+    focusedRangeStyle.left = getPositionByTimestamp({
+      timestamp: focusedIntervalStart,
+      timeIntervals,
+      zoomLevel,
+      wrapperWidth: timelineWrapperWidth
+    })
+    focusedRangeStyle.width = determineScaledWidth(
+      focusedIntervalEnd - focusedIntervalStart,
+      zoomLevel,
+      timelineWrapperWidth
+    ) + 2 // Accomodate for the 2px border on the interval
+
+    // Style the left focused interval mask
+    focusedRangeMaskLeftStyle.left = 0
+    focusedRangeMaskLeftStyle.width = determineScaledWidth(
+      focusedIntervalStart - timeIntervals[0],
+      zoomLevel,
+      timelineWrapperWidth
+    )
+
+    // Style the right focused interval mask
+    focusedRangeMaskRightStyle.left = getPositionByTimestamp({
+      timestamp: focusedIntervalEnd,
+      timeIntervals,
+      zoomLevel,
+      wrapperWidth: timelineWrapperWidth
+    }) + 2 // Accomodate for the 2px border on the interval
+    focusedRangeMaskRightStyle.width = determineScaledWidth(
```

Comment:
Add a line break?

Reply:
Looks like these are grouped together like the code above, it just looked a little weird because the `.left = ` line is multiline